rllib/core/rl_module/torch/torch_rl_module.py

```diff
@@ -54,6 +55,9 @@ def __init__(self, *args, **kwargs) -> None:
         # the interface of that base-class not the actual implementation.
         self.config = self.unwrapped().config
 
+    def get_action_dist_cls(self, *args, **kwargs) -> Type[TorchDistribution]:
```

Comment:
does it have to be *args, **kwargs? Why not inherit the signature?

Reply:
Passing through arguments makes it easier to hack the underlying get_action_dist_cls method, that's it.
The tradeoff is the missing signature.